apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -416,46 +421,84 @@ NetSimPacketEditor.prototype.makeBlurHandler = function (fieldName, converterFun
  */
 
 /**
+ * Convert binary to an integer, intentionally limiting the binary width so
+ * that overflow can occur.
+ * @param {string} binaryString (interpreted as unsigned)
+ * @param {number} maxWidth in bits
+ * @returns {number}
+ */
+var truncatedBinaryToInt = function (binaryString, maxWidth) {
+  return binaryToInt(binaryString.substr(-maxWidth));
+};
```

Comment:
Your preference for anonymous functions within blocks is [required by the Google style guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Function_Declarations_Within_Blocks#Function_Declarations_Within_Blocks) because regular function declarations in blocks are not _technically_ ECMAScript compliant and may misbehave on (very) bad implementations.

I've adopted declaring this way everywhere because in our pipeline everything is technically inside a block, and because it's visually more consistent with our method declarations.

I admit that, style-wise, this is probably the wrong approach - a style rule ought to apply to the form of the original source, not to the compiled product, right? - but I don't think browserify converts function declarations that are outside of blocks in the original source and get wrapped during concatenation, so it's kind of a style concession to a pipeline limitation.